server/services/interventionsService.ts

```diff
@@ -64,6 +64,16 @@ export interface CreateReportDateParams {
   toIncludingDate: CalendarDay
 }
 
+// Pagination parameters for resources that use spring pageable
+export interface PaginationParams {
+  // Page number to retrieve -- starts from 1
+  page?: number
+  // Number of elements in a page
+  size?: number
+  // Sort by property, defaults to ascending order. If descending is required then use ['$PROPERTY_NAME,DESC'] i.e. ['sentAt,DESC']
```

Comment:
small nit: the quotes aren't quite right in the example arrays `['sentAt', 'DESC']`

Reply:
or are they correct as they are? if so it might be worth clarifying why this param is a list

Reply:
DESC and ASC have to be coupled together with the sort property i.e. : ['sortByProperty1, DESC', 'sortByProperty2,ASC']

The documentation doesn't really explain that but it was what I observed from my testing.

Reply:
I've modified the comment to be more descriptive